src/framework/scene-registry.js

```diff
@@ -224,6 +224,7 @@ Object.assign(pc, function () {
     };
 
     /**
+     * @private
```

Comment:
I have a feeling @aidinabedi make this function public on the `pc.Application` API because engine-only devs need it. Any reason to make it private here?

Reply:
Ahh, interesting about making it public for engine devs (I see the original PR https://github.com/playcanvas/engine/issues/1944 )

Will have to rethink this.